src/containers/StoreContainer/StoreContainer.jsx

```diff
@@ -12,18 +12,42 @@ const StoreContainer = () => {
   const classes = useStyles();
   const store = useSelector(state => state.storeReducer.store);
   const [state, setState] = useState({
-    checkedA: false,
-    checkedB: false,
-    checkedC: false,
-    checkedD: false,
-    checkedE: false,
+    velas: false,
+    oraculos: false,
+    cursos: false,
+    cuarzos: false,
+    joyeria: false,
   });
+  const [filteredStore, setFilteredStore] = useState([])
   const dispatch = useDispatch();
 
   useEffect(() => {
     if (!store.length) dispatch(fetchStoreProducts());
   }, [dispatch, store]);
 
+  useEffect(() => {
+    if (Object.values(state).every((item => item === false))) {
+      setFilteredStore(store)
+    } else {
+      let categories = setCategories(state);
```

Comment:
change the function's name to avoid confusion with a use state hook, rename as getCategories or something like that